ngraph/frontend/onnx_import/src/utils/tensor_external_data.cpp

```diff
@@ -47,8 +47,10 @@ namespace ngraph
             {
 #if defined(ENABLE_UNICODE_PATH_SUPPORT) && defined(_WIN32)
                 std::wstring path = file_util::multi_byte_char_to_wstring(m_data_location.c_str());
+                file_util::sanitize_path(path, "\\.");
```

Comment:
It is implementation compatible with ONNX handling (`path.lstrip('/.')`), but I have a few doubts:
1. Correct path can be: /user/.onnx/weights.data -> current version won't handle it.
2. It doesn't block: ../weights.data case
Maybe better is: `sanitize_path(path, "..")`;?
or throwing exception if ".." is substring of the path? (with more user-friendly message)
@postrational What do you think?

Reply:
2nd option is better IMO

Reply:
I've improved `sanitize_path` and move handling to `update_external_data_paths`.